src/sentry/static/sentry/app/views/settings/incidentRules/ruleConditionsForm.tsx

```diff
@@ -138,7 +138,6 @@ class RuleConditionsForm extends React.PureComponent<Props, State> {
                     </SearchEventTypeNote>
                   </Tooltip>
                 }
-                help={t('Choose which metric to trigger on')}
```

Comment:
Was this intended to be a placeholder @EvanPurkhiser ?

Reply:
Yeah I couldn't find to be in use or implemented for searchbar, but it seems other fields accept it.